app/commands/exercise/representation/find_example_submissions.rb

```diff
@@ -0,0 +1,44 @@
+class Exercise::Representation::FindExampleSubmissions
+  include Mandate
+
+  initialize_with :representation
+
+  def call
+    example_submissions = [representation.source_submission]
+    example_submission_hashes = Set.new([solution_files_hash(representation.source_submission)])
+    page = 1
+
+    loop do
+      submissions = representation.
+        submission_representation_submissions.
+        where.not(id: representation.source_submission.id).
+        order(created_at: :desc).
+        page(page).
+        per(NUM_EXAMPLES)
```

Comment:
```suggestion
        per(NUM_EXAMPLES * 3)
```

Cheaper to return three times as many from one query than do three queries (presuming we think it'll take 3 queries to get the set). it could be 2x or 4x depending on the average.